aify/index.html

```diff
@@ -129,24 +145,167 @@ <h3 class="feature-title mt-0">Discover</h3>
+                                        </div>
+                                        <div class="feature-content">
+                                            <h4 class="feature-title mt-0">Individual Developers</h4>
+                                            <p class="text-sm mb-0">Lorem ipsum dolor sit amet, consecte adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua dui.</p> 
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="feature is-revealing">
+                                    <div class="feature-inner">
+                                        <div class="feature-icon">
+                                            <img class="asset-light" src="svg/feature-02-light.svg" alt="Feature 02">
+                                            <img class="asset-dark" src="svg/feature-02-dark.svg" alt="Feature 02">
+                                        </div>
+                                        <div class="feature-content">
+                                            <h4 class="feature-title mt-0">Medium scale business owners
+                                            </h4>
+                                            <p class="text-sm mb-0">Lorem ipsum dolor sit amet, consecte adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua dui.</p> 
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="feature is-revealing">
+                                    <div class="feature-inner">
+                                        <div class="feature-icon">
+                                            <img class="asset-light" src="svg/feature-03-light.svg" alt="Feature 03">
+                                            <img class="asset-dark" src="svg/feature-03-dark.svg" alt="Feature 03">
+                                        </div>
+                                        <div class="feature-content">
+                                            <h4 class="feature-title mt-0">Enterprise and Industries
+                                            </h4>
+                                            <p class="text-sm mb-0">Lorem ipsum dolor sit amet, consecte adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua dui.</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div> -->
+                            <div class="row">
+                                <div class="col-lg-4">
+                                    <div class="row zoom">
+                                        <img src="images/individual_developer.png" alt="" width="40%" id="like">
```

Comment:
this image is 135kb but according to its size of display it can probably be fit in 15-40kb..please optimize image